packages/paste-libraries/react-textarea-autosize/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@twilio-paste/react-textarea-autosize",
+  "version": "0.0.1",
```

Comment:
suggestion: I think what I've learnt is that starting at v0 should be reserved for alpha status, mainly due to what it does as a knock on effect when bumping package versions. This is essentially GA, so we should start at v1

Reply:
Ok! Will change it to 1.0.0